src/emcpy/stats.py

```diff
@@ -285,3 +286,45 @@ def bootstrap(insample, level=.95, estimator='mean', nrepl=10000):
     ci_upper = _np.percentile(deltas, upper_pctile)
 
     return ci_lower, ci_upper
+
+
+def spectrum_stats(df):
+    '''
```

Comment:
Do single quotes work to generate the automatic documentation?  I don't know the answer but I think, heretofore, double quotes have been used.  Maybe @CoryMartin-NOAA  can speak to that.